camp/schedule.html

```diff
@@ -1,115 +1,28 @@
 ---
-layout:     camp
+layout:     camp-detail
 title:      Offline Camp Schedule
 tab:        schedule
 root_path:  ../
 ---
+<div class="camp__wrapper">
 
-<header>
-  <h6>
-    <span class="month">November</span>
-    <span class="day">4-7<sup>th</sup></span>
-    <span class="year">2016</span>
-  </h6>
 
-  <p class="location">
-    <a href="https://goo.gl/maps/r4xFZGZ6kr12">Santa Margarita, California</a><br>
-    30 Campers
-  </p>
+  <h1>The Schedule</h1>
 
-  <hr>
+  <p>First things first. When booking your travel, please be careful to allow sufficient driving time to our somewhat remote destination. Hop over to our <a href="{{ page.root_path }}camp/faq.html#location">FAQ</a> for more info. </p>
 
-  <p class="price">
-    <small>
-      Registration has closed.
-    </small>
-  </p>
+  <p>Here's a quick look at what we have in store for you at Offline Camp. We'll share more detail as we get closer to the event.</p>
 
-</header>
-  
+  <h2>Friday, April 28 – Arrive and get comfy</h2>
+  <p>Campers may arrive at the ranch between 5:00 pm and 6:30 pm to start getting settled and exploring the property. Dinner will start promptly at 7:00 pm, followed by a welcome from your organizers.</p>
```

Comment:
not a ranch this time, it’s a castle! Maybe mansion would be more precise but castle sounds so cool :)